cmd/cloudflared/tunnel/cmd.go

```diff
@@ -546,7 +546,7 @@ func hostnameFromURI(uri string) string {
 	switch u.Scheme {
 	case "ssh":
 		return addPortIfMissing(u, 22)
-	case "rdp":
+	case "tcp":
 		return addPortIfMissing(u, 3389)
```

Comment:
This port, 3389, is for the RDP protocol: https://en.wikipedia.org/wiki/Remote_Desktop_Protocol

Reply:
Thanks for the PR, @mustafakirimli! Before we review the code (thanks @sodabrew!), do you mind creating an issue in the Github project outlining what this change is accomplishing and why?